integrations/auth0/client.go

```diff
@@ -22,6 +24,7 @@ var (
 	clientIDName     = sdktypes.NewSymbol("client_id")
 	clientSecretName = sdktypes.NewSymbol("client_secret")
 	domainName       = sdktypes.NewSymbol("auth0_domain")
+	authToken        = sdktypes.NewSymbol("oauth_AccessToken")
```

Comment:
```suggestion
	authTokenName        = sdktypes.NewSymbol("oauth_AccessToken")
```
I think it's confusing to name it `authToken` because it's not the token itself, it's just the symbol.
if you're going to add this, then I think you should also change line 74: 
```
token := vs.GetValueByString("oauth_AccessToken")
```
@daabr do we have a preference here?

Reply:
I agree with @pashafateev , but I don't have a specific recommendation. `XxxName` or `XxxVarName` seems reasonable.